packages/databricks-sdk-js/src/services/wsfs/WorkspaceFsDir.ts

```diff
@@ -77,6 +79,8 @@ export class WorkspaceFsDir extends WorkspaceFsEntity {
         @context ctx?: Context
     ) {
         const validPath = this.getAbsoluteChildPath(path);
+        // eslint-disable-next-line no-console
+        console.log(validPath);
```

Comment:
we should not use plain console.log from library code.

Reply:
removed